awx_collection/tests/integration/targets/workflow_job_template/tasks/main.yml

```diff
@@ -466,22 +466,23 @@
         workflow: "{{ wfjt_name }}"
         state: absent
       register: no_identifier_results
-      ignore_errors: true
+      ignore_errors: yes
 
     - assert:
         that:
-          - "no_identifier_results.msg == 'missing required arguments: identifier'"
+          - no_unified_results is defined
```

Comment:
seems this assert block is misplaced? needs to go under the "Fail if present and no unified job template set" task, as that is where "no_unified_results" is defined